lib/LINE/Bot/API.pm

```diff
@@ -619,6 +619,7 @@ https://opensource.org/licenses/Artistic-2.0
 =head1 SEE ALSO
 
 L<LINE::Bot::API::Event>,
-L<https://business.line.me/>, L<https://developers.line.me/>
+L<https://developers.line.biz/>
+L<https://at.line.me/en/>,
```

Comment:
Should we remove `en/`?

Reply:
`https://at.line.me/`
is auto-redirecting to 
`https://at.line.me/en/`